zpi-backend/src/main/java/pwr/zpibackend/services/user/AuthService.java

```diff
@@ -43,7 +43,7 @@ public Object getUserDetails(String email) {
         } catch (Exception ignored) {}
 
         if (employee != null && student != null) {
-            throw new EmployeeAndStudentWithTheSameEmailException(email);
+            throw new EmployeeAndStudentWithTheSameEmailException("Employee and student with the same email: " + email + " found");
```

Comment:
To nie zadziała dobrze, bo ten wyjątek akurat przyjmuje maila i wkłada go do wiadomości, więc albo musi zostać tu mail albo trzeba zmienić konstruktor wyjątku